tools/Raven.Debug/ThreadIo/ThreadsIoCommand.cs

```diff
@@ -0,0 +1,174 @@
+                                else if (line.StartsWith("syscr:"))
+                                {
+                                    ulong.TryParse(line.AsSpan("syscr:".Length).Trim(), NumberStyles.Integer, CultureInfo.InvariantCulture, out syscr);
+                                }
+                                else if (line.StartsWith("syscw:"))
+                                {
+                                    ulong.TryParse(line.AsSpan("syscw:".Length).Trim(), NumberStyles.Integer, CultureInfo.InvariantCulture, out syscw);
+                                }
+                            }
+
+                            var now = DateTime.UtcNow;
+                            if (prev.TryGetValue(tid, out var p))
+                            {
+                                var dt = Math.Max(1e-3, (now - p.TimestampUtc).TotalSeconds);
+                                var readOpsDelta = (long)((long)syscr - (long)p.Syscr);
+                                var writeOpsDelta = (long)((long)syscw - (long)p.Syscw);
+                                var readBytesDelta = (long)((long)readBytes - (long)p.ReadBytes);
+                                var writeBytesDelta = (long)((long)writeBytes - (long)p.WriteBytes);
+
+                                if (readOpsDelta < 0) readOpsDelta = 0;
+                                if (writeOpsDelta < 0) writeOpsDelta = 0;
+                                if (readBytesDelta < 0) readBytesDelta = 0;
+                                if (writeBytesDelta < 0) writeBytesDelta = 0;
+
+                                var opsDelta = readOpsDelta + writeOpsDelta;
+                                var bytesDelta = readBytesDelta + writeBytesDelta;
+
+                                var opsPerSec = opsDelta / dt;
+                                var mbPerSec = bytesDelta / (1024.0 * 1024.0) / dt;
+
+                                var opsReadPerSec = readOpsDelta / dt;
+                                var opsWritePerSec = writeOpsDelta / dt;
+                                var mbReadPerSec = readBytesDelta / (1024.0 * 1024.0) / dt;
+                                var mbWritePerSec = writeBytesDelta / (1024.0 * 1024.0) / dt;
+
+                                if (!totals.TryGetValue(tid, out var t))
+                                {
+                                    t = new ThreadIoTotals();
+                                    totals[tid] = t;
+                                }
+                                t.OpsTotal += opsDelta;
+                                t.MbTotal += bytesDelta / (1024.0 * 1024.0);
+                                t.ReadOpsTotal += readOpsDelta;
+                                t.WriteOpsTotal += writeOpsDelta;
+                                t.ReadMbTotal += readBytesDelta / (1024.0 * 1024.0);
+                                t.WriteMbTotal += writeBytesDelta / (1024.0 * 1024.0);
+
+                                var json = JsonSerializer.Serialize(new
```

Comment:
I wonder if we shouldn't have a switch to choose what format it will output. For internal usage JSON is nice but when using the tool from command line maybe we could format it differently?